modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/claimant_controller.rb

```diff
@@ -8,7 +8,7 @@ class ClaimantController < ApplicationController
       def search
         authorize nil, policy_class: ClaimantPolicy
 
-        poa_requests = policy_scope(PowerOfAttorneyRequest).joins(:claimant).where(claimant: { icn: })
+        poa_requests = policy_scope(PowerOfAttorneyRequest).joins(:claimant).where(claimant: { icn: }).not_withdrawn
```

Comment:
If we go with a (possibly temporary) solution to filter out withdrawn POA requests everywhere, we'd want to make that change in a more centrally located place so the logic applies for all the scenarios that correspond to surfacing POA requests.